frontend/pages/login.tsx

```diff
@@ -1,18 +1,27 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import useUser from '../src/auth/use-user';
 import { ErrorMapCtx, z, ZodIssueOptionalMessage } from 'zod';
 import { zodResolver } from '@hookform/resolvers/zod';
 import { useForm } from 'react-hook-form';
 import { TextFieldElement } from 'react-hook-form-mui';
 import { Box, Button, Paper, Typography } from '@mui/material';
 import { toast } from 'react-toastify';
+import { logger } from '../src/logger';
+import { UserRole } from '../src/models';
+import { useRouter } from 'next/router';
 
 export default function Login() {
+  const router = useRouter();
   // here we just check if user is already logged in and redirect to profile
-  const { mutateUser, user } = useUser({
-    redirectTo: '/search',
-    redirectIfFound: true
-  });
+  const { mutateUser, user } = useUser({});
```

Comment:
Dont need the empty object, should actually throw you an eslint error?

```suggestion
  const { mutateUser, user } = useUser();
```